src/main/java/io/kestra/plugin/airbyte/AbstractAirbyteConnection.java

```diff
@@ -87,6 +85,12 @@ protected <REQ, RES> HttpResponse<RES> request(RunContext runContext, MutableHtt
                 return client.toBlocking().exchange(request, argument);
             }
         } catch (HttpClientResponseException e) {
+            if (e.getStatus().getCode() == 409 && e.getResponse().getBody(String.class).isPresent()){
```

Comment:
Can really a 409 be thrown for something else?

Reply:
I don't know 😅

Reply:
Should I only test the Status Code ?

Reply:
If there is nothing clearly documented on theire side, better to keep it as is.